django_netjsongraph/tests/base/test_topology.py

```diff
@@ -370,16 +370,6 @@ def _assert_split_topology(self, topology):
             t.receive(network2)
             _assert_split_topology(self, t)
 
-    def test_very_long_addresses(self):
-        """
-        see https://github.com/netjson/django-netjsongraph/issues/6
-        """
-        t = self._set_receive()
-        data = self._load('static/very-long-addresses.json')
-        t.receive(data)
-        n = self.node_model.get_from_address('2001:4e12:452a:1:172::10', t)
-        self.assertEqual(len(n.addresses), 493)
```

Comment:
why are you removing this test?

Reply:
Oh my bad, I added it back and I convert it to string first because else it will give the length of array of addresses.